packages/server/test/course.integration.ts

```diff
@@ -71,4 +72,52 @@ describe('Course Integration', () => {
       });
     });
   });
+
+  describe('DELETE, /courses/:id/ta_location/:room', () => {
+    it('tests TA is deleted from queue if exists', async () => {
+      const ta = await UserFactory.create();
+      const queue = await QueueFactory.create({
+        room: 'The Alamo',
+        staffList: [ta],
+      });
+      const tcf = await TACourseFactory.create({
+        course: queue.course,
+        user: ta,
+      });
+
+      expect(
+        (await QueueModel.findOne({}, { relations: ['staffList'] })).staffList
+          .length,
+      ).toEqual(1);
+
+      await supertest({ userId: ta.id })
+        .delete(`/courses/${tcf.courseId}/ta_location/The Alamo`)
+        .expect(200);
+
+      expect(
+        await QueueModel.findOne({}, { relations: ['staffList'] }),
+      ).toMatchObject({
+        staffList: [],
+      });
+    });
+
+    it('tests error happens if ta not in queue', async () => {
+      const ta = await UserFactory.create();
+      const queue = await QueueFactory.create({ room: 'The Alamo' });
+      const tcf = await TACourseFactory.create({
+        course: queue.course,
+        user: ta,
+      });
+
+      await supertest({ userId: ta.id })
+        .delete(`/courses/${tcf.courseId}/ta_location/The Alamo`)
+        .expect(200);
```

Comment:
Title implies the response should return an error but the expected response code is a 200. Should the response be something like `404 Bad Request` instead?

Reply:
OOPS

Reply:
i think it might be better with a `4xx` but it doesn't actually impact us either way